bindings/python/pymongocrypt/mongocrypt.py

```diff
@@ -753,3 +792,23 @@ def __init__(self, ctx, filter, provider, master_key, callback):
             # Destroy the context on error.
             self._close()
             raise
+
+
+def _ask_for_kms_credentials(kms_providers):
+    """Get on-demand kms credentials.
+
+    This is a separate function so it can be overridden in unit tests."""
+    if 'aws' not in kms_providers:
+        return
+    if len(kms_providers['aws']):
+        return
+    if not _HAVE_AUTH_AWS:
+        raise RuntimeError(
+            "MONGODB-AWS authentication requires pymongo-auth-aws: "
```

Comment:
"MONGODB-AWS authentication" is not relevant here. Should say "on demand aws credentials require..."  

Should we add pymongo-auth-aws to pymongo[encryption] too?

Reply:
Opened https://github.com/mongodb/libmongocrypt/pull/452 and updated https://github.com/mongodb/mongo-python-driver/pull/1035